packages/common/src/hardhat/fixtures/default.ts

```diff
@@ -4,14 +4,32 @@ import { RegistryRolesEnum } from "../../Enums";
 import { CombinedHRE } from "../tasks/types";
 import { DeploymentsExtension } from "hardhat-deploy/types";
 
+const defaultFixtureWhitelist = [
```

Comment:
Not sure I like having deployments _not_ added to this whitelist by default. Seems like it could end up being error-prone. Maybe we could consider something less invasive as suggested below or maybe a blacklist?

Reply:
whitelisting will include less than blacklisting so hopefully will be lighter/faster.

If we did a blacklist, how would you like to construct this? ideally, we'd want a methods that can list all Tagged scripts and then we can iterate over them and remove them. I'll look into the hardhat settings to facilitate this.

Reply:
If we do this https://github.com/UMAprotocol/protocol/pull/3399#discussion_r718697665, can we just _not do either_ for now?

Reply:
Personally, I like the idea of deployment making all singleton contracts available, and if you need to do something more specialized, you write a special fixture for it, rather than making the default fixture opinionated. But I may be in the minority on that opinion :). Either way you decide to go is okay with me!